Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -311,15 +313,12 @@ extension GestureManager: GestureHandlerDelegate {
 
     // MapView has been panned
     internal func panned(by displacement: CGPoint) {
-        self.cameraManager.moveCamera(by: displacement, rotation: nil, pitch: nil, zoom: nil, animated: false)
+        self.cameraManager.moveCamera(by: displacement, rotation: nil, pitch: nil, zoom: nil, animated: false, didFling: false)
     }
 
     // Pan has ended on the MapView with a residual `offset`
     internal func panEnded(with offset: CGPoint) {
-        if let pitch = self.cameraManager.mapView?.pitch,
-           pitch == 0.0 {
-            self.cameraManager.moveCamera(by: offset, rotation: nil, pitch: nil, zoom: nil, animated: true)
-        }
+        self.cameraManager.moveCamera(by: offset, rotation: nil, pitch: nil, zoom: nil, animated: true, didFling: true)
```

Comment:
Is this always `true`? The pan can end without drift too.

Reply:
Yes, from what I saw in the existing behavior and my understanding of hydrogen, all panEnded has the potential to end with a drift and we should account for that.

Reply:
Hmm, really? Not sure if that's true. Would be worth a test.

Reply:
How would we test this?